examples/preprocessing/plot_power_transformer.py

```diff
@@ -0,0 +1,85 @@
+"""
+==========================================================
+Using PowerTransformer to apply the Box-Cox transformation
+==========================================================
+
+This example demonstrates the use of the Box-Cox transform
+through ``PowerTransformer`` to map data from various
+distributions to a Gaussian distribution.
+
+Box-Cox is useful as a transformation in modeling problems
+where homoscedasticity and normality are desired. Below
+are examples of Box-Cox applied to three distributions: a
+heavily left-skewed lognormal distribution, a left-skewed
+chi-squared distribution, and a right-skewed Weibull
+distribution.
+
+Note that when Box-Cox is applied to the lognormal distribution,
+the estimated parameter, lambda, takes on a value close to
+zero since Box-Cox is defined as the natural logarithm when
+lambda = 0. In all cases, the maximum likelihood estimated
+lambdas achieve very close results to the normal distribution.
+"""
+print(__doc__)
```

Comment:
I would print the doc after the import. I know that we are not checking the PEP8 but it does not cost anything to do so